troupon/deals/templates/deals/detail.html

```diff
@@ -2,7 +2,21 @@
 {% load static %}
 {% load humanize %}
 
-
+{% if deal %}
+{% block facebook_meta %}
```

Comment:
**Code Quality: Nitpick**

Mixed indentation. 2 spaces here and 4 spaces below.